server/docker-base/Dockerfile

```diff
@@ -349,13 +349,16 @@ RUN \
 # Switch SDK
 #
 
-ENV NINTENDO_SDK_ROOT ${PLATFORMSDK_DIR}/nx
-ENV SWITCH_SDK_FILENAME nx64-sdk-10.4.1.tar.gz
+ENV NINTENDO_SDK_ROOT_13_3_0 ${PLATFORMSDK_DIR}/nx-13.3.0
+# Backwards compatibility, since they're still using that env variable
+ENV NINTENDO_SDK_ROOT         ${NINTENDO_SDK_ROOT_13_3_0}
```

Comment:
The exception to the previous comment. However, since 10.4.1 isn't supported anymore, we had to update anyways.